osu.Game.Rulesets.Osu/Difficulty/Skills/OsuSkill.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using osu.Game.Rulesets.Difficulty.Preprocessing;
+using osu.Game.Rulesets.Difficulty.Skills;
+using osu.Game.Rulesets.Mods;
+using System.Linq;
+
+namespace osu.Game.Rulesets.Osu.Difficulty.Skills
+{
+    public abstract class OsuSkill : StrainSkill
+    {
+        public OsuSkill(Mod[] mods) : base(mods)
+        {
+        }
+
+        public double OsuDifficultyValue()
+        {
+            double difficulty = 0;
+            double weight = 1;
+
+            double strainMultiplier;
+            List<double> strains = GetCurrentStrainPeaks().OrderByDescending(d => d).ToList();
+
+            double baseLine = 0.68;
+
+            for (int i = 0; i <= 9; i++)
```

Comment:
I assume you only take 10 strains because of log10 scaling? Might wanna mention that in a comment

Reply:
I added parameters so it shouldn't be necessary now.